### DISCUSSION TITLE
Unsure how to fix broken profile pictures

### DISCUSSION THREAD

**Discussion by ShyViolets** [seerr-team/seerr#457] [Q&A] [answered]
Hello. I'm not sure what I've might've done incorrectly - but Jellyfin accounts in jellyseer have broken profile images even though they work fine in jellyfin. Was there some step I missed?

Info: Unraid 6.12.x, Jellyseer in docker, Jellyfin in Docker

**Comment by fallenbagel** [comment, marked as answer]
You need to add your external url to `external hostname` under settings > jellyfin. Then a simple log out and log back in should refresh your avatars

**Reply by ShyViolets** [reply]
Oh, I see. I guess since I haven't made JF available beyond my local network, I didn't have an external hostname. 

Thank you for your help and letting me know :)

**Reply by fallenbagel** [reply]
> Oh, I see. I guess since I haven't made JF available beyond my local network, I didn't have an external hostname. 
> 
> Thank you for your help and letting me know :)

If you are getting it broken in local network then you can add the local ipv4 of your machine with the port to it so all your lan devices get it fixed

**Comment by ShyViolets** [comment]
Just a headsup - I ended up putting in only my internal IP:Port and left external blank. After logging out/in - it seems the picture works now. Not sure about the other users - but just wanted to let you know.
